src/components/About/About.js

```diff
@@ -0,0 +1,15 @@
+import aboutTemplate from "./About.html";
+
+export class About {
```

Comment:
component name should be `AboutComponent`

Reply:
fixed